js/src/modals/DeployContract/DetailsStep/detailsStep.js

```diff
@@ -168,8 +188,12 @@ export default class DetailsStep extends Component {
             params.push('');
             break;
 
+          case 'bool':
```

Comment:
It is parameters according the the API, so it is consistent all the way through.